mount.go

```diff
@@ -111,6 +115,13 @@ func mount(source, destination, fsType string, flags int, options string) error
 		absSource = source
 	case typeTmpFs:
 		absSource = source
+	case typeHugeTlbfs:
+		absSource = source
+		//Allocate hugepages before mount
+		///sys/kernel/mm/hugepages/hugepages-1048576kB/nr_hugepages
+		///sys/kernel/mm/hugepages/hugepages-2048kB/nr_hugepages
+		//options eg "pagesize=2MB,size=107374182"
+		allocateHugePages(options)
```

Comment:
I imagine this could fail (and doesn't it return an error?) We should be looking to return an error here.